src/state/auth.js

```diff
@@ -30,7 +32,7 @@ class Auth {
         if (authResult && authResult.accessToken && authResult.idToken) {
           localStorage.setItem('accessToken', authResult.accessToken);
           localStorage.setItem('idToken', authResult.idToken);
-          history.push('/');
+          history.push(history.location.search.slice(6));
```

Comment:
use `new URLSearchParams(history.location.search).get('from')` to grab the `from` param value, this way it's a bit more deterministic than slicing whatever string may be in the search param